src/components/head/head.js

```diff
@@ -12,173 +12,186 @@ const Head = ({
-    />
-    <meta content="1024" name="twitter:image:width" />
-    <meta content="512" name="twitter:image:height" />
-    <meta
-      content={
-        imageUrl || `${siteUrl}/assets/Open-Graph-images-Website-Rules.jpg`
-      }
-      property="og:image"
-    />
-    <meta content="1024" property="og:image:width" />
-    <meta content="512" property="og:image:height" />
-
-    <meta content={themeColor} name="msapplication-TileColor" />
-    <meta
-      content="/ssw/include/pigeon/img/mstile-70x70.png"
-      name="msapplication-square70x70"
-    />
-    <meta
-      content="/ssw/include/pigeon/img/mstile-144x144.png"
-      name="msapplication-square144x144"
-    />
-    <meta
-      content="/ssw/include/pigeon/img/mstile-150x150.png"
-      name="msapplication-square150x150"
-    />
-    <meta
-      content="/ssw/include/pigeon/img/mstile-310x150.png"
-      name="msapplication-wide310x150"
-    />
-    <meta
-      content="/ssw/include/pigeon/img/mstile-310x310.png"
-      name="msapplication-square310x310"
-    />
-
-    <link href={`${pathPrefix}/manifest.json`} rel="manifest" />
-
-    <link
-      href="/ssw/include/pigeon/img/apple-touch-icon-57x57.png"
-      rel="apple-touch-icon"
-      sizes="57x57"
-    />
-    <link
-      href="/ssw/include/pigeon/img/apple-touch-icon-60x60.png"
-      rel="apple-touch-icon"
-      sizes="60x60"
-    />
-    <link
-      href="/ssw/include/pigeon/img/apple-touch-icon-72x72.png"
-      rel="apple-touch-icon"
-      sizes="72x72"
-    />
-    <link
-      href="/ssw/include/pigeon/img/apple-touch-icon-76x76.png"
-      rel="apple-touch-icon"
-      sizes="76x76"
-    />
-    <link
-      href="/ssw/include/pigeon/img/apple-touch-icon-114x114.png"
-      rel="apple-touch-icon"
-      sizes="114x114"
-    />
-    <link
-      href="/ssw/include/pigeon/img/apple-touch-icon-120x120.png"
-      rel="apple-touch-icon"
-      sizes="120x120"
-    />
-    <link
-      href="/ssw/include/pigeon/img/apple-touch-icon-144x144.png"
-      rel="apple-touch-icon"
-      sizes="144x144"
-    />
-    <link
-      href="/ssw/include/pigeon/img/apple-touch-icon-152x152.png"
-      rel="apple-touch-icon"
-      sizes="152x152"
-    />
-    <link
-      href="/ssw/include/pigeon/img/apple-touch-icon-180x180.png"
-      rel="icon"
-      sizes="180x180"
-      type="image/png"
-    />
-
-    <link
-      href="/ssw/include/pigeon/img/favicon-32x32.png"
-      rel="icon"
-      sizes="32x32"
-      type="image/png"
-    />
-    <link
-      href="/ssw/include/pigeon/img/favicon-16x16.png"
-      rel="icon"
-      sizes="16x16"
-      type="image/png"
-    />
-
-    <script type="application/ld+json">
-      {JSON.stringify(
-        schemaGenerator({
-          location,
-          canonical,
-          siteUrl,
-          pageTitle,
-          siteTitle,
-          pageTitleFull,
-        })
-      )}
-    </script>
-  </Helmet>
-);
+}) => {
+  const isHomePage = location.pathname === '/';
```

Comment:
I feel this might need an ? as location might be undefined during build 
```suggestion
  const isHomePage = location?.pathname === '/';
```

Reply:
well I guess it did build in staging... so maybe disregard these :P

Reply:
yes, the `location` is populated by Gatsby, so it should be present during build 🙂